godot-core/src/builtin/aabb.rs

```diff
@@ -82,6 +82,21 @@ impl Aabb {
     */
 }
 
+impl std::fmt::Display for Aabb {
+    /// Formats `Aabb` to match godot's display style.
```

Comment:
Please capitalize correctly: "Godot"

Also, "display style" is a bit vague, especially since there's nothing like a `Display` trait in Godot.
Maybe better:

```rs
    /// Formats `Aabb` to match Godot's string representation.
```

Same for the other occurrences.